reposilite-site/docs/reverse-proxy-ssl.md

```diff
@@ -0,0 +1,86 @@
+---
+id: reverse-proxy-ssl
+title: Reverse proxy ssl
+sidebar_label: Reverse proxy ssl
+---
+
+Lots of people like to use a reverse proxy like nginx with reposilite. This is a page on
+how to setup Nginx with SSL.
+
+Source: <https://gist.github.com/Saladoc/8c17bda7b131c4fade9b2189ea5050d8>
+
+## Step 1
+First, install and setup Reposilite. Make sure to setup Reposilite to listen on port 8080 (or anything other than 80 and 443).
+
+Then, install nginx, openssl, and certbot [using snapd](https://snapcraft.io/docs/installing-snapd/) `sudo snap install certbot --classic` and `sudo ln -s /snap/bin/certbot /usr/bin/certbot`
```

Comment:
Certbot is available with most package managers. Why specifically snapd? This shouldn't be presented as a hard requirement.

Reply:
That’s true, although certbot recommends snapd and doesn’t update their PPA IIRC.

Reply:
Wouldn't it be better to link to the certbot docs then as well?